src/current/v23.2/cockroach-debug-merge-logs.md

```diff
@@ -39,44 +40,48 @@ Generate a debug zip file:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug zip ./cockroach-data/logs/debug.zip --insecure
+cockroach debug zip ./cockroach-data/logs/debug.zip --insecure
 ~~~
 
 Unzip the file:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ unzip ./cockroach-data/logs/debug.zip
+unzip ./cockroach-data/logs/debug.zip
 ~~~
 
 Merge the logs in the debug folder:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug merge-logs debug/nodes/*/logs/*
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1
 ~~~
 
 Alternatively, filter the merged logs for a specified time range:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug merge-logs debug/nodes/*/logs/* --from="220713 18:36:28.208553" --to="220713 18:36:29.232864"
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1 --from="231031 19:19:50.917185" --to="231031 19:19:57.189263"
 ~~~
 
 You can also filter the merged logs for a regular expression:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cockroach debug merge-logs debug/nodes/*/logs/* --filter="RUNNING IN INSECURE MODE"
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1 --filter="ALL SECURITY CONTROLS HAVE BEEN DISABLED"
 ~~~
 
 You can redact sensitive information from the merged logs:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cockroach debug merge-logs --redact debug/nodes/*/logs/*
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1 --redact
 ~~~
 
+## Consideration
```

Comment:
This section should be called 'Considerations' (plural) to match our other pages (e.g. https://www.cockroachlabs.com/docs/v23.1/import-into#considerations)

Reply:
done